src/event-manager.js

```diff
@@ -196,6 +196,14 @@ export default class EventManager {
           if (enabled) {
             // Wait for this recognizer to fail
             otherRecognizer.requireFailure(name);
+            /**
+             * This seems to be a bug in hammerjs:
+             * requireFailure() adds both ways
+             * dropRequireFailure() only drops one way
+             * https://github.com/hammerjs/hammer.js/blob/master/src/recognizerjs/
+               recognizer-constructor.js#L136
+             */
+            recognizer.dropRequireFailure(otherName);
```

Comment:
Oof. Did you file a bug w/ hammer? Link here?

Reply:
The library does not seem to be actively maintained: https://github.com/hammerjs/hammer.js/issues/1197